tools/StaticAnalysis/Exceptions/AzureRM.IotHub/SignatureIssues.csv

```diff
@@ -13,3 +13,11 @@
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.SetAzureRmIotHub","Set-AzureRmIotHub","1","8410","Parameter RoutingProperties of cmdlet Set-AzureRmIotHub does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.SetAzureRmIotHub","Set-AzureRmIotHub","1","8410","Parameter Routes of cmdlet Set-AzureRmIotHub does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.SetAzureRmIotHubVerifiedCertificate","Set-AzureRmIotHubVerifiedCertificate","1","8420","Parameter set 'ResourceSet' of cmdlet 'Set-AzureRmIotHubVerifiedCertificate' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"D:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.IotHub\Microsoft.Azure.Commands.IotHub.dll","Microsoft.Azure.Commands.Management.IotHub.AddAzureRmIotHubRoute","Add-AzureRmIotHubRoute","1","8420","Parameter set 'ResourceSet' of cmdlet 'Add-AzureRmIotHubRoute' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
```

Comment:
These exceptions should no longer be needed since you changed the number of positional parameters for these cmdlets.

Reply:
still getting exception so didn't changed this file

Reply:
@anusapan You definitely shouldn't be getting these errors. Looking at both `AddAzureRmIotHubRoute` and `AddAzureRmIotHubRoutingEndpoint`, the highest position value you have is 2. So, I'd double check this. Also, if, for some reason, these exceptions need to remain, you have them duplicated. Meaning, the first 4 lines are the same as the last 4 lines.